esp-hal/src/soc/esp32/mod.rs

```diff
@@ -28,6 +28,8 @@ pub(crate) mod constants {
 
     pub const SOC_DRAM_LOW: u32 = 0x3FFA_E000;
     pub const SOC_DRAM_HIGH: u32 = 0x4000_0000;
+
+    pub const REF_TICK: fugit::HertzU32 = fugit::HertzU32::MHz(1);
```

Comment:
I wonder if these shouldn't go into `clock::Clocks` - on the other hand we already have a few frequencies here.

Most probably okay for now and we should rework the whole `clocks` thing since it's not very maintainable as it looks now 🤷‍♂️

Reply:
I thought about that too, but then I saw that it's used like twice in the whole driver (including this PR), so it was decided to leave it in the `soc` module for now. However, yes, after `clocks` thing is reworked, I guess we should consider having all the clock frequencies there

PS: Second place where `REF_TICK` freq is used: 
https://github.com/esp-rs/esp-hal/blob/main/esp-hal/src/clock/clocks_ll/esp32.rs#L6